nmrfx-utils/src/main/java/org/nmrfx/utils/TableUtils.java

```diff
@@ -0,0 +1,48 @@
+package org.nmrfx.utils;
+
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.input.Clipboard;
+import javafx.scene.input.ClipboardContent;
+import javafx.scene.input.DataFormat;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.IntStream;
+
+public class TableUtils {
+
+    private TableUtils() {}
+
+    /**
+     * Copies the contents of a table, including the headers, to the clipboard as a tab separated string.
+     * If no rows are selected or copyAll is true, then all rows will be copied. Otherwise only the
+     * selected rows will be copied.
+     * @param tableView The TableView to copy.
+     * @param forceCopyAll If true, all rows will be copied regardless of selections
+     */
+    public static void copyTableToClipboard(TableView<?> tableView, boolean forceCopyAll) {
```

Comment:
Used wildcard here so we could use this function for tables other than the Peak table.
Have tested with the peak table, region table, log table and change log level table